lighthouse-core/test/audits/dobetterweb/external-anchors-use-rel-noopener-test.js

```diff
@@ -88,19 +84,16 @@ describe('External anchors use rel="noopener"', () => {
     });
     assert.equal(auditResult.score, 0);
     assert.equal(auditResult.details.items.length, 2);
-    assert.equal(auditResult.details.items.length, 2);
+    assert.equal(auditResult.details.items[0].node.type, 'node');
```

Comment:
maybe we could get at least 1 assertion on the structure of the node value?

Reply:
I added the devtoolsNodePath to the AnchorElement mocks and added a couple assertions on the resulting item's node path. Not sure if this is what you meant though.